Jolt/Math/Vec3.inl

```diff
@@ -501,7 +501,9 @@ int Vec3::GetHighestComponentIndex() const
 
 Vec3 Vec3::Abs() const
 {
-#if defined(JPH_USE_SSE)
+#if defined(JPH_USE_AVX512)
+	return _mm_range_ps(mValue, mValue, 0b10'00);
```

Comment:
I don't use the ' separator in constants. Can you please remove it? (same comment applies to the other 3 locations where a ' is used)